doc/manual.rst

```diff
@@ -6692,16 +6692,16 @@ in C/C++).
 **Note**: This pragma will not exist for the LLVM backend.
 
 
-NoDecl pragma
+`nodecl` pragma
```

Comment:
If you're going to put this in backticks you should change all the pragma headers

Reply:
I've reverted that line to avoid blocking this PR; the backticks can be added in future work